readme.md

```diff
@@ -203,18 +203,11 @@ LTS releases may include additional extended support for security bug fixes as l
 
 This table shows previous LTS releases and end dates.
 
-| Release                                                                                                                        | LTS Start Date | Maintenance End Date | LTS End Date |
-| :----------------------------------------------------------------------------------------------------------------------------: | :------------: | :------------------: | :----------: |
-| [2022-01-18](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2021-3-18-patch4) <sub>patch 4 of 2021-03-18</sub> | 2022-01-18     | 2022-03-18           | 2024-03-17   |
-| [2021-10-19](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2021-3-18-patch3) <sub>patch 3 of 2021-03-18</sub> | 2021-10-19     | 2022-03-18           | 2024-03-17   |
-| [2021-8-12](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2021-3-18_patch2) <sub>patch 2 of 2021-03-18</sub>  | 2021-08-12     | 2022-03-18           | 2024-03-17   |
-| [2021-8-10](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2020-8-19_patch2) <sub>patch 2 of 2020-08-19</sub>  | 2020-08-10     | 2021-08-19           | 2023-08-19   |
-| [2021-6-23](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2021-3-18_patch1) <sub>patch 1 of 2021-03-18        | 2020-06-23     | 2022-03-18           | 2024-03-17   |
-| [2021-3-18](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2021-3-18)                                          | 2020-03-18     | 2022-03-18           | 2024-03-17   |
-| [2020-9-23](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2020-8-19_patch1) <sub>patch 1 of 2020-08-19</sub>  | 2020-09-23     | 2021-08-19           | 2023-08-19   |
-| [2020-8-19](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2020-8-19)                                          | 2020-08-19     | 2021-08-19           | 2023-08-19   |
-| [2020-4-3](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2020-1-31_patch1) <sub>patch 1 of 2020-01-31</sub>   | 2020-04-03     | 2021-01-30           | 2023-01-30   |
-| [2020-1-31](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2020-1-31)                                          | 2020-01-31     | 2021-01-30           | 2023-01-30   |
+| Release                                                                                                                        | LTS Start Date | Maintenance End Date |
+| :----------------------------------------------------------------------------------------------------------------------------: | :------------: | :------------------: |
+| [2021-3-18](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2021-3-18)                                          | 2021-03-18     | current              |
```

Comment:
This is the initial LTS release, we have patched it 4 times so the latest LTS release is
https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/2021_03_18-patch4

Reply:
We can potentially also add a link to the current LTS branch itself. Something to think about. For example, the latest LTS branch is :
https://github.com/Azure/azure-iot-sdk-csharp/tree/lts_2021_03

Reply:
Instead of linking the tags it would help if we linked the branches directly. There isn't an easy way to navigate from one tag to another (for the LTS patches) without going through the release page or finding a common link (through the tag prefix, etc.).

Reply:
I thought our plan was to not keep/maintain branches, rather to use tags.

Reply:
Oh, I thought we still had branches for LTS releases but all other non-LTS releases were off of tags.

Reply:
Yeah, we have branches (and tags) for LTS releases and only tags for all other releases.
![image](https://user-images.githubusercontent.com/22563986/160673680-904d82dc-897e-42a2-b163-563897e3e8d9.png)

Reply:
k

Reply:
will use tag with consistent tag name